examples/resnet_imagenet/README.md

```diff
@@ -56,21 +58,25 @@ If you're using the MosaicML platform, all you need to install is [`mcli`](https
 
 ```bash
 pip install --upgrade mosaicml-cli
+mcli init
 ```
 
 Then, just fill in a few fields in [yamls/mcloud_run.yaml](./yamls/mcloud_run.yaml):
 
 ```yaml
-cluster:   # Add the name of the cluster to use for this run
-gpu_type:   # Type of GPU to use; usually a100_40gb
-...
-  git_repo: mosaicml/examples  # Replace with your fork to use custom code
-  git_branch: main             # Replace with your branch to use custom code
+cluster: # Add the name of the cluster to use for this run
+gpu_type: # Type of GPU to use; usually a100_40gb
+---
```

Comment:
small detail, but is it safe to put `---` in the middle of a YAML file? Also, the lines below would need to be under a git integration and indented, I think?

Reply:
Yea good catch, not sure why it was ever like that 😛